spec/requests/legalaid-application_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe 'Legal aid applications' do
+  let(:response_json) {JSON.parse(response.body)}
+
+  describe  'POST /api/v1/applications' do
+    it 'returns an application reference' do
+
+      post '/api/v1/applications'
+      expect(response_json['data']['attributes']['application_ref']).to be
```

Comment:
I'm not sure that it is sufficient to test that the string you get is truthy. Won't this pass if `application_ref` is an empty string?

Maybe an approach would be to create two and ensure they have unique  `application_refs`?

Reply:
I thought that was not empty equivalent.  Although i think ```not_to be_empty``` is probably the better one to use here